tests/python/pants_test/engine/test_mapper.py

```diff
@@ -145,21 +145,33 @@ def unhydrated_structs(build_file_addresses):
   yield UnhydratedStructs(uhs)
 
 
-class AddressMapperTest(unittest.TestCase, SchedulerTestBase):
+class AddressMapperTest(TestBase, unittest.TestCase):
+  @classmethod
+  def extra_rules(cls):
+    # We add the `unhydrated_structs` rule because it is otherwise not used in the core engine.
+    return [unhydrated_structs]
+
   def setUp(self):
-    # Set up a scheduler that supports address mapping.
-    symbol_table = TargetTable()
-    address_mapper = AddressMapper(parser=JsonParser(symbol_table),
-                                   build_patterns=('*.BUILD.json',))
+    TestBase.setUp(self)
```

Comment:
I think the recommendation is to use `super(...).setUp()` because it only needs to refer to the name of this class, rather than needing edits when the inheritance hierarchy changes.